gui/src/selectors.js

```diff
@@ -28,3 +28,20 @@ export const jobRunsCountSelector = (state, jobSpecId) => {
   const spec = jobSpecSelector(state, jobSpecId)
   return spec ? spec.runsCount : 0
 }
+
+const camelCaseToUnderscore = str => (
+  str
```

Comment:
How do you feel about using a library like [`change-case`](https://github.com/blakeembrey/change-case) for this?

Reply:
Sure, I'm all for it. Didn't know `change-case` existed. I'll add it.